components/Tag.js

```diff
@@ -1,17 +1,41 @@
 import Link from "../components/Link";
 import { abbreviateNumber } from "../services/utils/abbreviateNumbers";
+import { useRouter } from "next/router";
+
+export default function Tag({ name, total, path,currentInput, selected, tags }) {
+  const router = useRouter();
+  const { search } = router.query;
+  const tagNames = []
+  tags && tags.slice(0,10).map(tag=> tagNames.push(tag.name))
 
-export default function Tag({ name, total }) {
   return (
     <div className="relative">
       <Link
-        href={`/search?search=${name}`}
-        className="flex flex-row p-1 m-2 rounded-lg text-sm font-mono border-2 cursor-pointer shadow-none hover:border-orange-600"
+        href={path && path ==='/search'?
```

Comment:
is it possible to extract it in a function rather than doing a large inline condition?. it is hard to know what's going on actually

Reply:
Ok I will try that once.

Reply:
Hey @ManishBisht777 I tried using a function but the href prop won't accept it. instead I have created variables for some part of the repetitive conditions so that it's a bit better to understand. It's working as usual.

Reply:
Also I noticed one problem which I don't understand if it's because of my code or something else but every time I write "ana" in search bar it gives some runtime error. This is not happening with any other user name or language as far as I know. Can u look into it because I am not getting any solution.

https://www.awesomescreenshot.com/video/14344045?key=b86391763fd9f830f3379a6c3d43d1f6

Reply:
Ah that is because one of the users with `ana` in their name is using an unsupported url for images - please do not worry about this, it is being fixed in another PR (it is not any of your changes)

Reply:
@eddiejaoude Okay that’s a relief. I thought I messed up something 😂